test/cql-pytest/test_keyspace.py

```diff
@@ -191,6 +191,59 @@ def creates(count):
             cql.execute(f"DROP KEYSPACE {keyspace}")
         cql.execute(f"CREATE KEYSPACE {keyspace} {ksdef}")
 
+# Test that passing "S3" parameters to storage options works and is stored
+# in a schema table
+def test_storage_options_s3(cql, scylla_only):
+    ksdef = "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : '1' } " \
+            "AND STORAGE = { 'type' : 'S3', 'bucket' : '42', 'endpoint' : 'localhost' }"
+    with new_test_keyspace(cql, ksdef) as keyspace:
```

Comment:
I don't understand something. When S3 **is** implemented, won't this creation of a keyspace try to do something on S3 - such as validating that the bucket exists, or creating an empty index file in the bucket, or something? If there is chance it will do this (is there?), this test will stop working. So isn't it safer to do this test with LOCAL instead of S3?

Reply:
That's a valid point, yes. I'll switch to local for now, and later I'll probably go with SHARED_FILESYSTEM, which is yet another backend that's already implemented in one of our hackathon branches, and which uses a local path (which could technically be a mounpoint for a distributed filesystem) for storing all the sstables.